devito/dle/rewriters.py

```diff
@@ -406,6 +405,29 @@ def _shm_parallelize(self, iet):
         """
         return self._shm_parallelizer.make_parallel(iet)
 
+    @dle_pass
+    def _hoist_prodders(self, iet):
```

Comment:
might it be a typo for `host`?

Reply:
nope, it's actually "hoist" in the sense of lifting